docs/DeveloperGuide.md

```diff
@@ -909,3 +909,88 @@ testers are expected to do more *exploratory* testing.
 
 1. Test case: `exit`<br>
        Expected: The app closes.
+
+## **Appendix: Planned Enhancements**
+
+Team size: 5
+
+1. **Enhancement 1**: Make deletion of suppliers with pending deliveries throw a warning and error message.
+   * **Description**: The current implementation allows users to delete suppliers with pending deliveries without any warning. 
+   * **Tasks**:
+     * We plan to add a warning message when a user tries to delete a supplier with pending deliveries.
+     * e.g. "Warning: Unable to delete supplier. There are pending deliveries associated with this supplier."
+
+
+2. **Enhancement 2**: Make marking an active supplier with pending deliveries inactive throw a warning and error message.
+   * **Description**: The current implementation allows users to mark active suppliers with pending deliveries as inactive without any warning.
+   * **Tasks**:
+     * We plan to add a warning message when a user tries to mark an active supplier with pending deliveries as inactive.
+     * e.g. "Warning: Unable to mark supplier as inactive. There are pending deliveries associated with this supplier."
+     
+
+3. **Enhancement 3**: Make improvements to detect duplicate suppliers. 
+   * **Description**: The current implementation defines duplicate suppliers as suppliers with the same name and company name.
+   However, it allows users to add duplicate suppliers with names with different case (e.g. "John Doe" and "john doe")
+   and multiple spacings (e.g. "John Doe" and "John   Doe").
+   * **Tasks**:
+     * We plan to redefine duplicate suppliers as suppliers with the same name and phone number. 
+     For the supplier name, we plan to make improvements to detect duplicate suppliers case-insensitively.
+     Supplier names with multiple spacings will also be considered as the same supplier.
+     Under this new implementation, suppliers are not allowed to have the same name and phone number. 
+     * e.g. "Warning: Duplicate supplier 'John Doe' already exists. Please use a different name."
```

Comment:
my interpretation was that we count them as duplicate suppliers is they have the same name (case insensitive) AND same number. so i don't think we the warning should be specific to either the name of the phone number as the we should still allow 2 'John Doe's with different phone number

Reply:
ohh ya make sense
would this be better? 
"Warning: Duplicate supplier detected. Please use a different name/phone number."

Reply:
yep that sounds good